src/storage/index.ts

```diff
@@ -30,18 +30,21 @@ export class Storage {
     this.encryption = encryption ? new WebCrypto() : new DummyEncryption()
   }
 
-  async store(order: Order): Promise<string> {
-    return this.backend.store(await this.encryption.encrypt(this.encode(order)))
+  async store(order: Order, publicKey: JsonWebKey = null): Promise<string> {
+    return this.backend.store(
+      await this.encryption.encrypt(this.encode(order), publicKey),
+    )
   }
-  async load(id: string): Promise<Order> {
+  async load(id: string, privateKey: JsonWebKey = null): Promise<Order> {
```

Comment:
Why is `privateKey` set to `null` by `default`? Does it make sense to call `encryption.decrypt` if `privateKey` is `null`?

Reply:
Only for the case that `Storage` was initialized with `encryption = false`, then no keys are needed of course. So that's mostly for local testing. I can also pass `null` in those tests. Wdyt is the cleaner / more beautiful way of doing this?

Reply:
Oh! I didn't know that's how you can do that in TS. Nice!